Test Failure: junit.framework.TestSuite.io.openliberty.microprofile.metrics.internal.fat.LibraryRefTest

RTC 301100

A Windows only problem

```io.openliberty.microprofile.metrics.internal.fat.LibraryRefTest:java.lang.RuntimeException: Error getting server NoMicrometerCore
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:206)
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:99)
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:81)
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:58)
at componenttest.rules.repeater.FeatureReplacementAction.setFeatures(FeatureReplacementAction.java:668)
at componenttest.rules.repeater.FeatureReplacementAction.setup(FeatureReplacementAction.java:508)
at componenttest.rules.repeater.JakartaEEAction.setup(JakartaEEAction.java:236)
at componenttest.rules.repeater.RepeatTests$CompositeRepeatTestActionStatement.evaluate(RepeatTests.java:144)
at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:364)
at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:178)
Caused by: componenttest.exception.TopologyException: (On Repeat Action _EE11_FEATURES_MicroProfile_70) Unable to delete pre-existing server directory: C:/Users/jazz_build/Build/workspace/ebcTestRunner/dev/autoFVT/image/output/wlp/usr/servers/NoMicrometerCore
at componenttest.topology.impl.LibertyServer.setup(LibertyServer.java:880)
at componenttest.topology.impl.LibertyServer.<init>(LibertyServer.java:820)
at componenttest.topology.impl.LibertyServerFactoryDelegate.newLibertyServer(LibertyServerFactoryDelegate.java:38)
at componenttest.topology.impl.LibertyServerFactoryDelegate.createLibertyServer(LibertyServerFactoryDelegate.java:33)
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:162)
```

--- a/dev/io.openliberty.microprofile.metrics.internal.5.1_fat/fat/src/io/openliberty/microprofile/metrics/internal/fat/LibraryRefTest.java
+++ b/dev/io.openliberty.microprofile.metrics.internal.5.1_fat/fat/src/io/openliberty/microprofile/metrics/internal/fat/LibraryRefTest.java
@@ -146,7 +146,7 @@ public class LibraryRefTest {
 	public void noMicrometerCore() throws Exception {
 		server = serverNoMicrometerCore;
 		server.startServer();
-
+		server.waitForSSLStart();
 		// CWMMC0014I emits that metrics is using libraryRef
 		Assert.assertNotNull("CWMMC0014I Not found",
 				server.waitForStringInLogUsingMark("CWMMC0014I"));

